libraries/AC_Avoidance/AC_Avoid.h

```diff
@@ -146,8 +158,11 @@ class AC_Avoid {
     *        It then calculates the desired backup velocity and passes it on to "find_max_quadrant_velocity" method to distribute the velocity vector into respective quadrants
     * OUTPUT: The method then outputs four velocities (quad1/2/3/4_back_vel_cms), which correspond to the final desired backup velocity in each quadrant
     */
-    void calc_backup_velocity(float kP, float accel_cmss, Vector2f &quad1_back_vel_cms, Vector2f &qua2_back_vel_cms, Vector2f &quad3_back_vel_cms, Vector2f &quad4_back_vel_cms, float back_distance_cm, Vector2f limit_direction, float dt);
-
+    void calc_backup_velocity_2D(float kP, float accel_cmss, Vector2f &quad1_back_vel_cms, Vector2f &qua2_back_vel_cms, Vector2f &quad3_back_vel_cms, Vector2f &quad4_back_vel_cms, float back_distance_cm, Vector2f limit_direction, float dt);
+    /*
+    * Compute the back away velocity required to avoid breaching margin, including vertical component
+    */
```

Comment:
we should probably put some more comments here especially for the arguments that are new compared with the calc_backup_velocity_2D.  So I guess min_z_vel and max_z_vel are the return vertical velocities to push the vehicle either up (i.e. away from obstacles below) or down (i.e. away from obstacles above).

Reply:
we might want to be consistent in the argument naming between this function and "find_max_quadran_velocity_3D".  I guess "z_up" is the same as "min_z_vel"?  Maybe we should clarify if "z_down" is a positive or negative number when descending?